src/test/regress/expected/multi_extension.out

```diff
@@ -966,14 +966,15 @@ SELECT * FROM multi_extension.print_extension_changes();
 -- Snapshot of state at 11.0-1
 ALTER EXTENSION citus UPDATE TO '11.0-1';
 SELECT * FROM multi_extension.print_extension_changes();
-                           previous_object                            | current_object
+                           previous_object                            |                          current_object
```

Comment:
Is style difference expected?

Reply:
I did not expect this to change. However the tests pass on my branch, so it should be ok.

I do not understand the reason why we have this change, however I am quite sure that it is ok.